composer.json

```diff
@@ -15,17 +15,21 @@
         "symfony-cmf/routing-auto": "^2.0",
         "symfony-cmf/routing-bundle": "^2.0",
         "aferrandini/urlizer": "1.0.*",
-        "phpdocumentor/reflection-docblock": "^3.1, !=3.2.0"
+        "phpdocumentor/reflection-docblock": "^3.1, !=3.2.0",
+        "symfony-cmf/core-bundle": "^2.0",
```

Comment:
we should have the check using the translatable interface also check if interface_exists instead of making this a hard requirement.

Reply:
It is a good idea, but I am sorry, I don't know how to change https://github.com/symfony-cmf/routing-auto-bundle/pull/223/files#diff-2fe84fe19eb48e039a7f135b5fb1f07fR76 to use the string